gas-bound-caller/contracts/GasBoundCaller.sol

```diff
@@ -46,9 +45,7 @@ contract GasBoundCaller {
         // This require is more of a safety protection for the users that call this function with incorrect parameters.
         //
         // Ultimately, the entire `gas` sent to this call can be spent on compute regardless of the `_maxTotalGas` parameter.
-        if (_maxTotalGas < gasleft()) {
-            revert InsufficientGas();
-        }
+        require(_maxTotalGas >= gasleft(), "Gas limit is too low");
```

Comment:
Here we're going back from custom errors?

Reply:
yes, it is needed to maintain the same codehash for gas bound caller